lib/ui/painting.dart

```diff
@@ -4367,41 +4379,109 @@ enum VertexMode {
 }
 
 /// A set of vertex data used by [Canvas.drawVertices].
+///
+/// Vertex data consists of a series of points in the canvas coordinate space.
+/// Based on the [VertexMode], these points are interpreted either as
+/// independent triangles ([VertexMode.triangles]), as a sliding window of
+/// points forming a chain of triangles each sharing one side with the next
+/// ([VertexMode.triangleStrip]), or as a fan of triangles with a single shared
+/// point ([VertexMode.triangleFan]).
```

Comment:
Probably worth incorporating two points into this:

* triangleFan is no longer natively supported by most backends, meaning it must be emulated by "unrolling" to VertexMode.triangles
* triangleFan is also not particularly efficient even when it is natively supported because structurally it has more fragments that have to be evaluated and thrown away along the triangle intersections. This point is maybe less important than the former.

Reply:
added:

```
  /// This mode is not natively supported by most backends, and is instead 
  /// implemented by unrolling the points into the equivalent 
  /// [VertexMode.triangleStrip], which is generally more efficient. 
```

Reply:
Actually on impeller I unroll it to Triangles, TriangeStrip unrolling requires even more work 😅

Reply:
oops, misread your comment

Reply:
fixed

Reply:
LGTM!